docs/installation/ca/Installing_CA_with_Secure_Database_Connection.md

```diff
@@ -6,62 +6,32 @@ Overview
 
 This page describes the process to install a CA subsystem with a secure database connection.
 
-Ensure that the secure connection has been enabled on the directory server.
-Export the signing certificate for the directory server into ds_signing.crt.
-This step is described [here](https://www.dogtagpki.org/wiki/DS_SSL).
+Before beginning with the installation, please ensure that you have configured the directory
+server and added base entries.
+The step is described [here](https://github.com/dogtagpki/pki/wiki/DS-Installation).
+
+Export the DS signing certificate for the directory server into `ds_signing.crt`.
```

Comment:
When I first read "Export the DS signing certificate ..."  I began to wonder whether I missed the instruction on how to generate the "DS signing certificate".  It wasn't till after I clicked into the linked [Export DS certificates] did I realize that the certs were genererate at installation.  Perhaps, preface this paragraph with the same opening statement in the linked doc, "By default DS will generate a self-signed CA certificate and a sever certificate signed by the CA certificate."